src/shared_utilities/models.jl

```diff
@@ -291,6 +294,67 @@ end
 
 Domains.coordinates(model::AbstractModel) = Domains.coordinates(model.domain)
 
+
+"""
+"""
+function add_dss_buffer_to_aux(
+    p::ClimaCore.Fields.FieldVector,
+    domain::Domains.AbstractDomain,
+) end
+
+function add_dss_buffer_to_aux(
+    p::ClimaCore.Fields.FieldVector,
+    domain::Union{Domains.Plane, Domains.SphericalSurface},
+)
+    keys = propertynames(p)
```

Comment:
@akshaysridhar do you know of a better way of appending another element to a FieldVector? this doesnt work as is...and is also kind of clunky

Reply:
Does `p` need to be a `FieldVector` here ? Or could it be a general cache container ?